js/src/elements/form-behavior.ts

```diff
@@ -379,8 +359,7 @@ class FormBehaviorElement extends HTMLElement {
 				continue;
 			}
 
-			const isTagSelector = input.classList.contains('tag-input');
-			if (input.hasAttribute('aria-hidden') && !isTagSelector) {
+			if (input.hasAttribute('aria-hidden')) {
```

Comment:
I feel like this is work for another time, but should we explicitly check if `aria-hidden` is `"true"`? It seems to me that `aria-hidden="false"` would trip this up.

Reply:
Good point. Let's table this for a follow up PR so it won't block the integration into Docs.

Reply:
Makes sense to me!